error when linking for wownerujo

running `./gradlew build` for wownerujo, with wownero wallet lib on master d8606ef:

```
undefined reference to `Monero::WalletImpl::defaultMixin() const'
```

PR coming

--- a/src/wallet/api/wallet.h
+++ b/src/wallet/api/wallet.h
@@ -143,8 +143,6 @@ public:
     virtual Subaddress * subaddress();
     virtual SubaddressAccount * subaddressAccount();
     virtual void setListener(WalletListener * l);
-    virtual uint32_t defaultMixin() const;
-    virtual void setDefaultMixin(uint32_t arg);
     virtual bool setUserNote(const std::string &txid, const std::string &note);
     virtual std::string getUserNote(const std::string &txid) const;
     virtual std::string getTxKey(const std::string &txid) const;

--- a/src/wallet/api/wallet2_api.h
+++ b/src/wallet/api/wallet2_api.h
@@ -696,16 +696,6 @@ struct Wallet
     virtual Subaddress * subaddress() = 0;
     virtual SubaddressAccount * subaddressAccount() = 0;
     virtual void setListener(WalletListener *) = 0;
-    /*!
-     * \brief defaultMixin - returns number of mixins used in transactions
-     * \return
-     */
-    virtual uint32_t defaultMixin() const = 0;
-    /*!
-     * \brief setDefaultMixin - setum number of mixins to be used for new transactions
-     * \param arg
-     */
-    virtual void setDefaultMixin(uint32_t arg) = 0;
 
     /*!
      * \brief setUserNote - attach an arbitrary string note to a txid

